blackjack/blackjack_test.go

```diff
@@ -33,23 +34,37 @@ func TestNewBlackjackGame(t *testing.T) {
 		Cards: stack,
 	}
 
+	setupOutput := &bytes.Buffer{}
+	setupInput := strings.NewReader("1\nPlanty\na\n3")
+
 	g, err := blackjack.NewBlackjackGame(
 		blackjack.WithCustomDeck(deck),
+		blackjack.WithOutput(setupOutput),
+		blackjack.WithInput(setupInput),
 	)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	g.Start()
-
-	got := blackjack.ReportMap[g.Player.HandOutcome]
+	blackjack.RunCLI(g)
```

Comment:
It doesn't feel quite right to have to create the game and then pass it to `RunCLI`. In general, a function like `RunCLI` is intended to eliminate all paperwork—for example, we'd expect it to create the game object itself, with the appropriate inputs and outputs for a CLI.